### PR TITLE
[Example] MNIST inference after training offloading

### PR DESCRIPTION
**Pull request by songgot**
[Example] MNIST inference after training offloading
    
 Example of performing inference after training offloading consists of two apps: sender and receiver.
    
    The scenario is as follows
    1. sender: offloading MNIST training data to the receiver
    2. receiver: train the model with offloaded data
    3. receiver: offloading the trained model to the sender and register it in the sender's ml model repo
    4. sender: receives a model and performs inference using the offloaded model
    
Fixing ML API bug, changing to renamed APIs, and using the new ML option will be updated by the new PR with the new rootstrap released by the official

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]

File: Tizen.native/MNIST_inference_after_training_offloading/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+input_layers=flatten
+Unit = 10                           # Output Layer Dimension ( = Weight Width )
+weight_initializer = xavier_uniform
+bias_initializer = zeros
+Activation = softmax                # activation : sigmoid, softmax
+```
+
+#### 2. Model offloading ####
+To offload the trained model, the roles of sender and receiver are reversed.
+The receiver(model sender) and the sender(model receiver) create ```ml_option_h``` and set ```dest-host```, ```dest-port```, ```connect-type```, ```host```, and ```topic``` to connect nnstreamer-edge network. And both sender and receiver create remote servece with ```ml_remote_service_create``` for model offloading.
```
Comment (gichan-jang) [inline Tizen.native/MNIST_inference_after_training_offloading/README.md:127]: Please update `ml_remove_service_*` to `ml_service_remote_*`.

File: Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
+    ad->option_h = NULL;
+  }
+  if (ad->service_option_h) {
+    ml_option_destroy (ad->service_option_h);
+    ad->service_option_h = NULL;
+  }
+  /** bug
+     if (ad->service_h)
+     ml_service_destroy (ad->service_h);
+   */
```
Comment (gichan-jang) [inline Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c:91]: Is this still a bug? Hasn't it been fixed yet?

File: Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c [hunk 3]
```diff
@@ -0,0 +1,844 @@
...
+      ad->transmission_model);
+
+  ret = g_file_get_contents (ad->transmission_model, &ad->contents, &len, NULL);
+  if (FALSE == ret) {
+    dlog_print (DLOG_ERROR, LOG_TAG, "Failed to get contents(%s)",
+        ad->transmission_model);
+    return;
+  }
+
+  g_usleep (1000000);
```
Comment (gichan-jang) [inline Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c:521]: Let's make sleep time more readable or add comments.

File: Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c (hunk 2 above)
Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c:91]: I will fix this in the next PR with a new rootstrap.

File: Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c (hunk 3 above)
Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/receiver/src/receiver.c:521]: I will fix this in the next PR with a new rootstrap.

File: Tizen.native/MNIST_inference_after_training_offloading/README.md (hunk 1 above)
Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/README.md:127]: Fixing ML API bug, changing to renamed APIs, and using the new ML option will be updated by the new PR with the new rootstrap released by the official. :)
Thank you.

Comment (gichan-jang) [reply Tizen.native/MNIST_inference_after_training_offloading/README.md:127]: Ok, Let's update it later.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**